### ISSUE TITLE
Contrast issues in dark mode

### ISSUE THREAD

**Issue by plannero** [edrlab/readium-desktop#3324]
There are a number of texts that become almost impossible to read when the application is in "dark mode", due to insufficient contrast.

See screen dumps from search function and bookmarks list: 

<img width="313" height="453" alt="screen dumps from search function and bookmarks list" src="https://github.com/user-attachments/assets/0feac598-18d0-4170-92a3-fe16cf26b856" />

Using Thorium v3.3.0 under MacOS.

**Comment by danielweck** [comment]
wow that's really bad. it's strange because I personally always test with my default system dark mode (MacOS) and other developers test in their preferred light mode.

**Comment by danielweck** [comment]
could you please clarify if your operating system is in light or dark mode?

**Comment by gautierchomel** [comment]
I can reproduce on MacOS with the system in light mode and Thorium in dark mode. That does not happens if the system is also in dark mode.

**Comment by plannero** [comment]
Application in dark mode, OS in light mode, yes.
